Autocoders/Python/src/fprime_ac/utils/version.py

```diff
@@ -0,0 +1,14 @@
+""" fprime version handling and reporting """
+import os
+from setuptools_scm import get_version
+
+ROOT_PARENT_COUNT=5
+
+def get_fprime_version():
+    """ Gets the fprime version using setuptools_scm  """
+    # First try to read the SCM version
+    try:
+        return get_version(root=os.sep.join([".."] * ROOT_PARENT_COUNT), relative_to=__file__)
+    # Fallback to a specified version when SCM is unavailable
+    except LookupError:
+        return "1.5.4" # Must be kept up-to-date when tagging
```

Comment:
Is it possible to move this version number to an easy to update place so it's hard for us to forget?

Reply:
I'd like to leave it here for now.  I have a plan that will make bumping it irrelevant.

Reply:
Here is the short version of the plan:

1. Create F´ metapackage on pypi
2. Have the above package write out a version file to be included above.
3. Ship the above output as our release tarball.

All automatic, no hand editing.  ....but it is more than I'd like to fit into this PR.

Reply:
I'm okay with leaving this as is as long as there is a plan for future automation.